### PR TITLE
fix alert for expiring credits

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/orgs/models.py [hunk 1]
```diff
@@ -881,8 +896,9 @@ def get_low_credits_threshold(self):
                                     self._calculate_low_credits_threshold)
 
     def _calculate_low_credits_threshold(self):
-        last_topup = self.topups.filter(is_active=True).last()
-        return int(last_topup.credits * 0.25)
+        now = timezone.now()
```
Comment (norkans7) [inline temba/orgs/models.py]: here I consider low threshold 25% of all active topups available

File: temba/orgs/models.py [hunk 2]
```diff
@@ -866,9 +868,22 @@ def get_credits_expiring_soon(self):
...
+
+        used_credits = TopUpCredits.objects.filter(topup__in=expiring_topups_qs).aggregate(Sum('used')).get('used__sum')
+
+        expiring_topups_credits = expiring_topups_qs.aggregate(Sum('credits')).get('credits__sum')
+
+        more_valid_credits_qs = self.topups.filter(is_active=True, expires_on__gt=one_month_period)
+        more_valid_credits = more_valid_credits_qs.aggregate(Sum('credits')).get('credits__sum')
+
+        if more_valid_credits or not expiring_topups_credits:
+            return 0
```
Comment (norkans7) [inline temba/orgs/models.py]: no alert if more active credits available or no expiring credits in the next month

**Comment by norkans7** [comment]
Please review

File: temba/orgs/models.py (hunk 1 above)
Comment (nicpottier) [inline temba/orgs/models.py]: Let's do 15% on this.. 25% might be a bit early..

**Comment by nicpottier** [comment]
Should merge master into this so that the tests pass but looks good!

**Pull request merged**